rebar.lock

```diff
@@ -1,100 +0,0 @@
-{"1.2.0",
```

Comment:
What for you delete lock file?

Reply:
To recreate it and get the latest references. We will need a proper ergw_aaa tag first, btw :-/